documentation/modules/ROOT/pages/operations/debezium-server.adoc

```diff
@@ -305,9 +306,64 @@ When the alternative implementations are not available then the default ones are
 |`@CustomConsumerBuilder`
 |Custom configured instance of a `KinesisClient` used to send messages to target streams.
 
-|[[kinesis-ext-stream-name-mapper]]<<kinesis-ext-stream-name-mapper, `io.debezium.server.kinesis.StreamNameMapper`>>
+|[[kinesis-ext-stream-name-mapper]]<<kinesis-ext-stream-name-mapper, `io.debezium.server.StreamNameMapper`>>
 |
-|Custom implementation maps the planned destination (topic) name into a physical Kinesis stream name. By default the same name is used.
+|Custom implementation maps the planned destination (topic) name into a physical Kinesis stream name.
+By default the same name is used.
+
+|=======================
+
+
+==== Google Pub/Sub
+
+Google Pub/Sub is an implementation of messaging/eventing system designed for scalable batch and stream processing applications.
+Pub/Sub exposes a set of REST APIs and provides a (not-only) Java SDK that is used to implement the sink.
+
+[cols="35%a,10%a,55%a",options="header"]
+|=======================
+|Property
+|Default
+|Description
+
+|[[pubsub-type]]<<pubsub-type, `debezium.sink.type`>>
+|
+|Must be set to `pubsub`.
+
+|[[pubsub-project-id]]<<pubsub-projectid, `debezium.sink.pubsub.projectid`>>
+|_system-wide default project id_
+|A project name in which the target topics are created.
+
+|[[pubsub-ordering]]<<pubsub-ordering, `debezium.sink.pubsub.ordering.enabled`>>
```

Comment:
Re ordering, I found that one: https://cloud.google.com/pubsub/docs/ordering. Should we link it here? I haven't full understood it yet, could you perhaps give a summary in layman's terms?

Reply:
https://googleapis.dev/java/google-cloud-clients/latest/index.html?com/google/cloud/pubsub/v1/Publisher.html

Reply:
Worth adding a link to that to the docs, I think.